04_health/exposure-to-trauma/exposure-to-trauma-2020.Rmd

```diff
@@ -0,0 +1,434 @@
+  filter(!is.na(county_name))
+```
+
+### Join 2022 CHR data and 2016-2020 CDC WONDER data
+```{r}
+# Join our 2020 data to CHR 2020 data
+chr_test <- left_join(x = chr2022,
+                      y = deaths %>%
+                        filter(year==2020) %>%
+                        select(GEOID, deaths, population, rate_injury_death_crude, rate_injury_death),
+                      by = "GEOID") %>%
+  mutate(rate_injury_death_chr = round(rate_injury_death_chr, 1))
+
+```
+
+### Compare values from CHR and CDC WONDER
+
+Visually inspecting the data shows that, for most observations, the rate produced by CHR matches the crude rate from CDC WONDER. There are 161 observations that do not match. These are observations where the rate was `unreliable` because the death count was less than 20. CHR produced the crude rate by dividing the death count by population for these counties and multiplying by 100,000. This computation does not work for us because we are reporting age-adjusted rates, not crude rates.
+```{r}
+# Check differences between CHR rates and CDC crude rates
+test <- chr_test %>%
+  filter(is.na(rate_injury_death_crude) & !is.na(rate_injury_death_chr))
+
+```
+
+```{r}
+rm(test, chr_test, chr2022)
+```
+
+
+### Visualizations of CDC WONDER data
+```{r}
+deaths %>%
+  mutate(size = if_else(population < 200000, "1. small population", "2. big population")) %>%
+  ggplot(aes(population, rate_injury_death)) +
+  geom_point(alpha = 0.1) +
+  facet_wrap(~ size, scales = "free_x") +
+  labs(title = "High Injury Death Rates are Concentrated in Smaller Counties") +
+  scatter_grid()
+
+```
+
+```{r}
+deaths %>%
+  ggplot(aes(deaths, rate_injury_death)) +
+  geom_point(alpha = 0.2) +
+  labs(title = "High Injury Death Rates are Concentrated in Counties with Few Injury Deaths") +  
+  scatter_grid()
+
+```
+
+
+## 5. Save the data
+This section reads in the Urban Institute's universe of counties, joins the CDC injury death data to those counties, and writes out the file.
+
+Read in Urban county file for 2014-2020
+```{r}
+all_counties <- read.csv(here::here("04_health", "county-populations.csv")) %>%
+  filter(!year %in% c(2021, 2022)) %>%
+  mutate(state = str_pad(state, 2, pad = "0"),
+         county = str_pad(county, 3, pad = "0"),
+         GEOID = str_c(state, county))
+
+```
+
+16 observations (4 counties) are in our universe of counties but not in the CDC data.
+```{r}
+missing_CDC <- anti_join(x = all_counties, y = deaths, by = "GEOID")
+
+missing_CDC %>%
+  group_by(state_name, county_name) %>%
+  count()
+
+```
+
+49 observations (7 counties) are in the CDC data but not our universe of counties.
+```{r}
+missing_urban <- anti_join(x = deaths, y = all_counties, by = "GEOID")
+
+missing_urban %>%
+  group_by(county_name) %>%
+  count()
+
+```
+
+Data for Bedford city, VA (FIPS code 51515) are only available for 2013 and earlier; for computational purposes, counts and population estimates for 2014 and later are set to zero. 
```

Comment:
I moved the section that reads in Urban's universe of counties and joins the CDC WONDER onto it to the beginning. The missing death rates in this section aren't available because these counties no longer exist as such and have been included in the data only for consistency. I included more context explaining this and have removed them from the data.